### DISCUSSION TITLE
How to define a panoramic fisheye camera?

### DISCUSSION THREAD

**Discussion by TaoHuUMD** [facebookresearch/pytorch3d#728] [Ideas]
## 🚀 Feature

NOTE: Please look at the existing list of Issues tagged with the label ['enhancement`](https://github.com/facebookresearch/pytorch3d/issues?q=label%3Aenhancement). **Only open a new issue if you do not see your feature request there**.

## Motivation

It would be great if I can define a panoramic fisheye camera. How to define a fisheye camera in the current version?

## Pitch

NOTE: we only consider adding new features if they are useful for many users.

**Comment by nikhilaravi** [comment]
@TaoHuUMD for new feature requests and "how-tos" please use the "Discussions" tab in GitHub: https://github.com/facebookresearch/pytorch3d/discussions

To create a new camera class you can inherit from the [`CamerasBase`](https://github.com/facebookresearch/pytorch3d/blob/master/pytorch3d/renderer/cameras.py#L20) class and define the necessary methods.
